### PR TITLE
Fix Minerva LX `info.json`

### PR DESCRIPTION
**Pull request by Gondolindrim**
## Description

Fixes the info.json layout of minerva LX and adds missing column key defs

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/archetype/minervalx/info.json
Comment (zvecr) [inline keyboards/archetype/minervalx/info.json:1]: This file should not be renamed back to `info.json`

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/archetype/minervalx/info.json
Comment (Gondolindrim) [reply keyboards/archetype/minervalx/info.json:1]: What should it be named?

Comment (zvecr) [reply keyboards/archetype/minervalx/info.json:1]: `keyboard.json`, like it is currently in develop.

Comment (Gondolindrim) [reply keyboards/archetype/minervalx/info.json:1]: Done in [09c8d80](https://github.com/qmk/qmk_firmware/pull/23706/commits/09c8d80279a132f1ab71f0f6ef2e937eb7db1857)

**Review by drashna — APPROVED** [review]

**Review by zvecr — CHANGES_REQUESTED** [review]
Build failures need to be resolved,
```
./keyboards/archetype/minervalx/keymaps/default/keymap.c:27:5: error: macro "LAYOUT" requires 70 arguments, but only 65 given
```

**Pull request closed without merging**